collections/high_availability/roles/drbd/README.md

```diff
@@ -62,6 +131,38 @@ drbdadm status
 drbdmon
 ```
 
+Example:
+```
+[root@mycluster ~]# drbdadm status
+resource0 role:Primary
+  disk:UpToDate
+  bee-meta2 role:Secondary
+    peer-disk:UpToDate
+
+resource1 role:Primary
+  disk:UpToDate
+  bee-meta2 role:Secondary
+    peer-disk:UpToDate
+```
+
+
+
+
+To delete the configuration (**will delete all data on the DRBD devices**)
+run the following commands :
+
+```
+drbdadm down resource0;drbdadm invalidate resource0;drbdadm wipe-md resource0 && rm -f /etc/drbd.d/resource0.res
+drbdadm down resource1;drbdadm invalidate resource1;drbdadm wipe-md resource1 && rm -f /etc/drbd.d/resource1.res
+rm -f /etc/drbd.d/global_common.conf
+```
+
+## Documentations
+
+https://computingforgeeks.com/install-and-configure-drbd-on-centos-rhel/
+https://linbit.com/drbd-user-guide/drbd-guide-9_0-en/#p-apps
+
 ## Changelog
 
+* 1.0.1: Role improvements. Hamid Merzouki <hamid.merzouki@naverlabs.com>
```

Comment:
You did more than fixing issues. Can you increase version to 1.1 ?

Reply:
ok, I will increase to 1.1
thanks